packages/firestore/src/protos/firestore_proto_api.ts

```diff
@@ -224,6 +224,15 @@ export declare namespace firestoreV1ApiClientInterfaces {
   interface ExistenceFilter {
     targetId?: number;
     count?: number;
+    unchangedNames?: BloomFilter;
+  }
+  interface BloomFilter {
+    bits?: BitSequence;
+    hashCount?: number;
+  }
+  interface BitSequence {
+    bitmap?: string | Uint8Array;
```

Comment:
Ahh I see where this is decoded: https://github.com/firebase/firebase-js-sdk/blob/e9fb4037f3908d2a05070de9219aa8a94d287f14/packages/firestore/src/remote/serializer.ts#L239-L259

This confirms that `string | Uint8Array` is the correct type.

You can resolve this comment once you read it. I'm just including it here for completeness.

Reply:
nice! Thank you Denver. Xijiao had mentioned that the backend will send a string, but I like to keep it as string | Uint8Array